### DISCUSSION TITLE
Error when using "navigate" with esm bundling

### DISCUSSION THREAD

**Discussion by thomas-serverhouse** [livewire/livewire#6144] [7. Bugs (archived)]
### Livewire version

3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Install a new Laravel 10 and Livewire 3 beta
```
laravel new livewire-esm
composer require livewire/livewire "^3.0@beta"
npm i
npm run dev
```
2. In app.js import Livewire esm 
```js
// app.js
import { Livewire } from '../../vendor/livewire/livewire/dist/livewire.esm';
Livewire.start();
```
3. Create a few dummy routes in web.php
```php
// web.php
Route::get('/', function () {
    return view('welcome', ['name' => 'Home']);
});
Route::get('/page1', function () {
    return view('welcome', ['name' => 'Page 1']);
});
Route::get('/page2', function () {
    return view('welcome', ['name' => 'Page 2']);
});
```
4. Update welcome.blade.php with the following code
```html
<!DOCTYPE html>
<html>
    <head>
        @vite('resources/js/app.js')
    </head>
    <body>

        <div>
            <a href="/" wire:navigate>Home</a>
            <a href="/page1" wire:navigate>Page 1</a>
            <a href="/page2" wire:navigate>Page 2</a>
        </div>

        <h1>{{$name}}</h1>

        @livewireScriptConfig
        <script type="text/javascript">
            console.log(location.href);
        </script>
    
    </body>
</html>
```
5. Now display the page in the browser and try to navigate, the console will show an error

### Additional information

If i remove the type="text/javascript" attribute in my blade file the problem is fixed. 
However using plugins that add a script tag will result in an error like laravel's debugbar for exemple `composer require barryvdh/laravel-debugbar --dev`.

This problem seems to be fixed if we simply add a missing "let" in the livewire.esm.js file
```js
function cloneScriptTag(el) {
    // ....
   for (let attr of el.attributes) {
        script.setAttribute(attr.name, attr.value);
   }
}
```
I can't tell if "attr" is supposed to be a global variable or not.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PhiloNL** [comment]
Thanks for reporting. I've submitted a fix: https://github.com/alpinejs/alpine/pull/3685
